test/test_transforms.py

```diff
@@ -82,7 +82,7 @@ def test_mu_law_companding(self):
         self.assertTrue(waveform_exp.min() >= -1. and waveform_exp.max() <= 1.)
 
     def test_scriptmodule_AmplitudeToDB(self):
-        spec = torch.rand((6, 201))
```

Comment:
Reason to change the test?

Reply:
Sorry, that is plain old poor git management. yikes. In its defence, however, it tests for batch dimension.

Reply:
Just checking :)

Reply:
Also, removed or not, this change is completely irrelevant to the GPU script changes.

Reply:
Reverted.